corehq/apps/hqwebapp/middleware.py

```diff
@@ -2,18 +2,13 @@
 
 from django.middleware.csrf import CsrfViewMiddleware, REASON_NO_CSRF_COOKIE, REASON_BAD_TOKEN
 from django.conf import settings
-from django.contrib import messages
-from django.utils.translation import ugettext as _
-
-from corehq.util.soft_assert import soft_assert
 
 logger = logging.getLogger('')
 
 
 class HQCsrfViewMiddleWare(CsrfViewMiddleware):
 
     def _reject(self, request, reason):
-        messages.error(request, _('If cookies are enabled and you still see this error, please report an issue'))
         request_url = request.path
         referring_url = request.META.get('HTTP_REFERER', 'Unknown URL')
         warning = "Request at {url} doesn't contain a csrf token. "\
```

Comment:
I think this warning message is not correct anymore. On the next line it says "Letting the request pass through for now." which I think is not true unless `CSRF_SOFT_MODE` is enabled.